content/about.md

```diff
@@ -3,15 +3,15 @@ title: About us
 image: images/profile.png
 ---
 ## About us
-DevRel Collective exists to facilitate sharing information, resources, and encouragement among the Developer Relations community.
+The DevRel Collective exists as a place for DevRel professionals, Community Managers, and others to share resources, learn best practices, support one another, and be amongst our peers. We participate in and nurture communities as our jobs, but we, too need a community of our own. This is that place. We are over **1,800 professionals** (and growing all the time!) from all walks of life and all over the world.
 
-If you share a passion for the Developer community, and you would like to get to know a great group of friendly peers, reach out about joining the collective.
+If you're a DevRel Professional and are looking for your people, here we are! Join us!
 
 ### Why is this an exclusive group?
-We feel it’s important for those of us who are pursuing this career to have a private and safe place to talk about our struggles, offer suggestions and congratulations, seek advice, and create professional relationships with others in the industry. As a result, we are limiting access to this group to those who are actively entrenched in a career in Developer Relations.
+We feel it’s important for those of us who are pursuing this career to have a private and safe place to talk about our struggles, offer suggestions and congratulations, seek advice, and create professional relationships with others in the industry. As a result, we limit access to this group to those who are actively engaged in a career in Developer Relations or Community Management.
```

Comment:
I would clarify "Community Management for a technical community" so that there isn't any confusion about other types of communities.